blog/2023-05-11-conda-doctor.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "conda doctor : Detecting conda environment inconsistencies easily"
+slug: "2023-05-11-conda-doctor"
+description: |
+    The new `conda doctor` command allows users to detect packages with missing files in their conda environments.
+
+authors: [forgottenprogramme]
+tags: [conda, environments]
+
+---
+The newest `conda doctor` command will be available with the latest conda 23.5.0, enabling conda users to detect corrupted packages (packages with files missing) in their conda environment quickly.
```

Comment:
I'm also thinking the intro should start with a "so what?" sentence first. Here's a possibility that would cover the entire intro:

"One of the more tedious problems that can happen to a conda environment is when package files are accidentally deleted. The package becomes corrupt, your project throws errors, and you have to painstakingly, manually search for the missing files. That is, until now. The new `conda doctor` command will be available with conda version 23.5.0 and enable conda users to quickly detect corrupted packages (packages with files missing) in their conda environment. Instead of manually checking for missing files, with `conda doctor`, users can learn about all corrupted packages in their conda environment with a single, short command."

I'm not sure if "project" is the right term above. I've never had a missing package file, so I'm not terribly familiar. Does the IDE throw errors when the user attempts to run a script or does conda itself complain at them?